downstream/modules/analytics/proc-create-savings-plan.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+// assembly-automation-savings-planner.adoc
+
+
+[id="proc-create-savings-plan_{context}"]
+
+= Creating a new automation savings plan
+
+Create an automation savings plan by defining the tasks needed to complete an automation job using the {planner}.
+
+. Navigate to *Red Hat Insights* > *Savings Planner*.
+. Click *Add Plan*.
+. Provide information about your automation job using the provided form:
+.. Enter descriptive information, such as a name, description, and type of automation.
+.. Enter technical information, such as the number of hosts, the duration to manually complete this job, and how often you complete this job.
+.. Click *Next*.
+
+[NOTE]
```

Comment:
This note may be better placed by taking the elements mentioned and moving them up to introductory sentence. Maybe as bullet points for visibility?